public/protocols/development.netcanvas/protocol.json

```diff
@@ -154,6 +154,11 @@
             "label": "Prompt id",
             "description": "Prompt id",
             "type": "text"
+          },
+          "special_category": {
+            "label": "Special Category",
+            "description": "Special Category",
+            "type": "categorical"
```

Comment:
Shouldn't the categorical variable define its valid categories here, same as an ordinal variable?

Reply:
Bryans PR adds this so just merging them in sequence.